### ISSUE TITLE
No focus boxes shown for Olympus E-5 and OM-D E-M5II

### ISSUE THREAD

**Issue by Subhash-at** [musselwhizzle/Focus-Points#129]
For the Olympus E-5 and also for E-M5 Mark II there are no error reports in the dialog windows but also no focus boxes shown in the last window shown by the plugin after calculation.

**Comment by musselwhizzle** [comment]
I heard that Olympus updated the software for certain cameras which broken this plugin. Attach an image from each camera to this ticket and I'll take a look when I have time (wont be for a while).

**Comment by Subhash-at** [comment]
[Joshua Musselwhite <notifications@github.com> schrieb am 17.9.2017 um 11:02 Uhr:]

>I heard that Olympus updated the software for certain cameras which
>broken this plugin. Attach an image from each camera to this ticket and
>I'll take a look when I have time (wont be for a while).

Hi!

Thank you for this great offer! Attached two images with EXIF data.

Kind regards
Subhash

**Comment by FarFromSane** [comment]
Hello
I also have this issue on my Olympus EM1.  It has Firmware 4 on it.   
In my case.. the red focus box is always displayed in the upper left corner
I tried to attach some examples but they were to large.   
Let me know where to send them.

**Comment by Subhash-at** [comment]
It is easy to create smaller images with the EXIF data in them: Just reduce the size temporary in the camera and take a picture, then choose your preferred size again. The data size of this picture will be small too. If you choose a subject which has big clear areas in it the compression will be more effective and the image data size will be even smaller. (I took a picture of a black plank before a white background.)

**Comment by FarFromSane** [comment]
I had been attempting to upload Raw.    I have attached 2 zip files with jpegs.  
Archiv.zip is from an OMD EM-5 (original OMD.  No new Firmware in quite a long time)
Archive2 is from an OMD EM-1 with Firmware 4 
I also attach a screenshot showing where the plugin placed the focus box (upper left) and an arrow showing where the real one is.
<img width="923" alt="screen shot 2017-09-25 at 9 25 44 am" src="https://user-images.githubusercontent.com/15947708/30810772-259b119c-a1d4-11e7-9947-7ed2d11d5957.png">
[Archive 2.zip](https://github.com/musselwhizzle/Focus-Points/files/1329632/Archive.2.zip)
[Archive.zip](https://github.com/musselwhizzle/Focus-Points/files/1329633/Archive.zip)

**Comment by musselwhizzle** [comment]
@Subhash-at - Looking at this ticket, your files dont seem to be there. Could you try again please

@FarFromSane - I opened your jpeg from Archive.zip and the focus point shows entirely different to me. Perhaps you need to update to the latest version of this plugin. See image below: 

<img width="864" alt="screen shot 2017-10-11 at 7 44 29 pm" src="https://user-images.githubusercontent.com/2308674/31476739-e61b046e-aebc-11e7-9af0-f232ceaa3fa4.png">

**Comment by Subhash-at** [comment]
Second try.
![om-d_e-m5ii_179190132](https://user-images.githubusercontent.com/12187069/31480673-69e6066a-af1f-11e7-9929-f78e318a7577.JPG)
![e-5_179199915](https://user-images.githubusercontent.com/12187069/31480699-9432794e-af1f-11e7-8d3d-7c5521bef552.JPG)

**Comment by musselwhizzle** [comment]
@Subhash-at 
See attached. If this is correct, please update to the latest version of the plugin. 

<img width="866" alt="screen1" src="https://user-images.githubusercontent.com/2308674/31528800-5b99ce1e-af8a-11e7-929c-b8d15db6ef57.png">
<img width="876" alt="screen2" src="https://user-images.githubusercontent.com/2308674/31528801-5bd52bd0-af8a-11e7-8525-0834191b1e69.png">

**Comment by Subhash-at** [comment]
This is correct. I updated but **the point is not shown in Lightroom 5.7.** Then I came up with the idea that a newer Lightroom version may do it and I tried it on my laptop **with Lightroom 6.0. And there it works!** See attachments:

![bildschirmfoto-2017-10-13-um-10 25 54](https://user-images.githubusercontent.com/12187069/31537319-f69dbfd4-b001-11e7-968b-a0c01290cde6.jpg)
![bildschirmfoto-2017-10-13-um-10 20](https://user-images.githubusercontent.com/12187069/31537320-f6c52538-b001-11e7-9242-d8f583deb84e.jpg)

**Comment by Subhash-at** [comment]
Another point to consider as an user:

I'm quite sure that I did not focus on the bush in the middle ground but on the grass in front. If I did afterwards I turned the camera to set my frame. But this cannot be pictured by any plugin and also not in the camera. The focus point is correct but it was located on an other part of the subject.

**Comment by FarFromSane** [comment]
Thanks for the tip about LR 5 vs 6.  
I'm on LR 5.7 .. that is likely why it's not working.   
I"m upgrading this weekend and will post back here with results.

**Comment by musselwhizzle** [comment]
Interesting that there's a difference between LR 5 and 6. Below is the only place in the code where we make a distinction between the 2 versions. Maybe the code isn't accurate there
https://github.com/musselwhizzle/Focus-Points/blob/master/focuspoints.lrdevplugin/DefaultPointRenderer.lua#L268

Unfortunately, I dont have LR5 or an olympus camera, so other people contributed to doing the work here so unfortunately it looks like I won't be much help. If you guys can code, I'd be happy to have you look at the files here to make sure they are behaving correctly.

**Comment by Subhash-at** [comment]
Also having no experience with lua-scripts I tried to find the error. I looked at the code, changed some lines, but had no success. Sorry. Also the debug log was not useful for me.
